package.json

```diff
@@ -19,9 +19,10 @@
     "postexample-generate": "npm --prefix ./tram-one-example install",
     "example-build": "npm --prefix ./tram-one-example run build",
     "example-start": "npm --prefix ./tram-one-example start",
-    "example-start-and-kill": "(npm run example-start) & (sleep 5 && npm run example-check) & (sleep 8 && pkill webpack-serve)",
+    "example-start-and-kill": "(npm run example-start) & (sleep 5 && npm run example-check) && (npm run kill-webpack)",
```

Comment:
by making this `&& (npm run kill-webpack)`, we'll kill the webpack instance as soon as our tests finish.